.github/actions/setup-sentry/action.yml

```diff
@@ -56,6 +56,7 @@ runs:
         [ "$GITHUB_REF" = "refs/heads/master" ] && echo "PYTEST_SENTRY_ALWAYS_REPORT=1" >> $GITHUB_ENV || true
 
         ### services configuration ###
+        echo "BIGTABLE_EMULATOR_HOST=localhost:8086" >> $GITHUB_ENV
```

Comment:
Does having this on trigger the tests to run?

Reply:
Yes: https://github.com/getsentry/sentry/blob/270627fe7957f83fa225e027db5f85365383ac4f/tests/sentry/nodestore/bigtable/backend/tests.py#L75-L78